File: php/src/Algorithm/Fizzbuzz.php
<?php

namespace App\Algorithm;


class Fizzbuzz
{

    function fizzBuzz(int $limit): void
    {
        for ($i = 1; $i <= $limit; $i++) {

            if ($i % 3 == 0 && $i % 5 == 0) {
                echo "FizzBuzz\n";
            } elseif ($i % 3 == 0) {
                echo "Fizz\n";
            } elseif ($i % 5 == 0) {
                echo "Buzz\n";
            } else {
                echo $i . "\n";
            }
        }
    }
}

File: golang/fizzbuzz/fizzbuzz.go
package main

import "fmt"

func main(){

	fizzbuzz(15)
}


func fizzbuzz(n int)  {

	for i:=1; i<=n; i++ {
		switch  {
		case i%3 == 0 && i%5 == 0:
			fmt.Println("FizzBuzz")
		case i%3 == 0:
			fmt.Println("Fizz")
		case i%5 == 0:
			fmt.Println("Buzz")
		default:
			fmt.Println(i)			
		}
	}

}